src/System Application/Test Library/Email/src/ConnectorMock.Codeunit.al

```diff
@@ -69,6 +70,28 @@ codeunit 134688 "Connector Mock"
         EmailRateLimit.Insert();
     end;
 
+    procedure AddAccountv2(var EmailAccount: Record "Email Account")
```

Comment:
Create an overload of the old function isntead `AddAccount` that takes in the connector. Remove duplicate code